Library/Homebrew/cmd/audit.rb

```diff
@@ -610,7 +610,7 @@ def audit_text
       problem "\"Formula.factory(name)\" is deprecated in favor of \"Formula[name]\""
     end
 
-    if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin]
+    if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin] && text !~ %r[kibana]
```

Comment:
@DomT4 I was thinking the same but the `return` here would mean other audits wouldn't run, unfortunately.